### PR TITLE
Add s390x architecture support

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
Add s390x support since the build issues in nixpkgs are resolved now.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
Added s390x architecture support to static binaries.
```

### CODE REVIEW

File: scripts/release-notes/release_notes.go [hunk 1]
```diff
@@ -139,6 +139,13 @@ Download one of our static release bundles via our Google Cloud Bucket:
   - [cri-o.ppc64le.%s.tar.gz.spdx](https://storage.googleapis.com/cri-o/artifacts/cri-o.ppc64le.%s.tar.gz.spdx)
   - [cri-o.ppc64le.%s.tar.gz.spdx.sig](https://storage.googleapis.com/cri-o/artifacts/cri-o.ppc64le.%s.tar.gz.spdx.sig)
   - [cri-o.ppc64le.%s.tar.gz.spdx.cert](https://storage.googleapis.com/cri-o/artifacts/cri-o.ppc64le.%s.tar.gz.spdx.cert)
+- [cri-o.s390x.%s.tar.gz](https://storage.googleapis.com/cri-o/artifacts/cri-o.s390x.%s.tar.gz)
```
Comment (kwilczynski) [inline scripts/release-notes/release_notes.go:142]: Whites paces issue?

Comment (saschagrunert) [reply scripts/release-notes/release_notes.go:142]: Hm, in which sense?

Comment (kwilczynski) [reply scripts/release-notes/release_notes.go:142]: Ah, no. This is a new section. I check the whole file now. Looked like something was wrong with an indent.

Sorry for the commotion! Doh!

**Pull request merged**